Source/Scene/Cesium3DTileset.js

```diff
@@ -1286,309 +1470,276 @@ define([
-                                        // and this tile can refine to them.
-                                        touch(tileset, child, outOfCore);
-                                    }
-                                }
-                            }
-                        } else if (useChildrenBoundUnion) {
-                            ++stats.numberOfTilesCulledWithChildrenUnion;
-                        }
-                    } else {
-                        // Tile does not meet SSE and its children are loaded.  Refine to them in front-to-back order.
-                        childrenVisibility = computeChildrenVisibility(t, frameState, true);
-                        for (k = 0; k < childrenLength; ++k) {
-                            child = children[k];
-
-                            if (isVisible(child.visibilityPlaneMask)) {
-                                stack.push(child);
-                            } else {
-                                // Touch the child tile even if it is not visible. Since replacement refinement
-                                // requires all child tiles to be loaded to refine to them, we want to keep it in the cache.
-                                touch(tileset, child, outOfCore);
-                            }
-                        }
+        var childrenVisibility = computeChildrenVisibility(tile, frameState, true);
+        var child, i;
 
-                        if (childrenVisibility & Cesium3DTileChildrenVisibility.VISIBLE_IN_REQUEST_VOLUME) {
-                            t.replaced = true;
-                            if (defined(t.descendantsWithContent)) {
-                                scratchRefiningTiles.push(t);
-                            }
-                        } else if (!useChildrenBoundUnion || (childrenVisibility & Cesium3DTileChildrenVisibility.VISIBLE)) {
-                            // Even though the children are all loaded they may not be visible if the camera
-                            // is not inside their request volumes.
-                            selectTile(tileset, t, fullyVisible, frameState);
-                        } else if (useChildrenBoundUnion) {
-                            ++stats.numberOfTilesCulledWithChildrenUnion;
-                        }
+        var maximumScreenSpaceError = tileset._maximumScreenSpaceError;
+        var showAdditive = tile.refine === Cesium3DTileRefine.ADD && tile._sse > maximumScreenSpaceError;
+        var showReplacement = tile.refine !== Cesium3DTileRefine.ADD && (childrenVisibility & Cesium3DTileChildrenVisibility.VISIBLE_IN_REQUEST_VOLUME) !== 0;
+
+        if (showAdditive || showReplacement) {
+            for (i = 0; i < childrenLength; ++i) {
+                child = children[i];
+                if (isVisible(child.visibilityPlaneMask)) {
+                    if (!showAdditive || getScreenSpaceError(tileset, tile.geometricError, child, frameState) > maximumScreenSpaceError) {
```

Comment:
Also, this sse is different. It's computed using the parent's geometric error.

https://github.com/AnalyticalGraphicsInc/cesium/blob/3d-tiles/Source/Scene/Cesium3DTileset.js#L1358